lxc/image.go

```diff
@@ -425,7 +425,7 @@ func (c *imageCmd) run(config *lxd.Config, args []string) error {
 		} else {
 			progress := ProgressRenderer{Format: i18n.G("Transferring image: %s")}
 			handler := func(percent int64, speed int64) {
-				progress.Update(fmt.Sprintf("%d%% (%s/s)", percent, shared.GetByteSizeString(speed)))
+				progress.Update(fmt.Sprintf("%d%% (%s/s)", percent, shared.GetByteSizeString(speed, -1)))
```

Comment:
I know this is minor, but is there any reason to pass -1 here and correct it in the function? Can we just make the argument a uint and pass 2 everywhere?